FirebaseRemoteConfig/Sources/RCNConfigFetch.m

```diff
@@ -75,6 +78,7 @@ @implementation RCNConfigFetch {
   NSURLSession *_fetchSession;  /// Managed internally by the fetch instance.
   NSString *_FIRNamespace;
   FIROptions *_options;
+  NSString *_templateVersionNumber;
```

Comment:
any particular reason `templateVersionNumber` is a string and not a number?

Reply:
It's returned as a string in the fetch response and so we keep it like that and then convert it later in the Realtime client when we need it for version checking.